lib/employers.ts

```diff
@@ -2,5 +2,6 @@
 const employers = new Array();
```

Comment:
export const employers = [{id: "10", username: "elodie"}, /*...*/]

// never export default if you can